photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java

```diff
@@ -76,12 +76,12 @@ public enum PoseStrategy {
     /**
      * Create a new PhotonPoseEstimator.
      *
-     * @param fieldTags A WPILib {@link AprilTagFieldLayout} linking AprilTag IDs to Pose3ds with
-     *     respect to the FIRST field.
+     * @param fieldTags A WPILib {@link AprilTagFieldLayout} linking AprilTag IDs to Pose3d objects
```

Comment:
link to wpilib field coordinate system page. also dont know why there is the abbreviation